data/src/main/java/me/androidbox/data/di/AuthenticationModule.kt

```diff
@@ -0,0 +1,16 @@
+package me.androidbox.data.di
+
+import dagger.Binds
+import dagger.Module
+import dagger.hilt.InstallIn
+import dagger.hilt.android.components.ViewModelComponent
+import me.androidbox.data.remote.authentication.AuthenticationRepositoryImp
+import me.androidbox.domain.authentication.remote.AuthenticationRepository
+
+@InstallIn(ViewModelComponent::class)
+@Module
+interface AuthenticationModule {
+
+    @Binds
+    fun bindsAuthenticationImp(authenticationImp: AuthenticationRepositoryImp): AuthenticationRepository
```

Comment:
I would rename it to: bindsAuthenticationRepository
Since that's what it actually binds